### DISCUSSION TITLE
clarification on updateTime and validTimes

### DISCUSSION THREAD

**Discussion by dangelsaurus** [weather-gov/api#607] [Q&A] [answered]
I've noticed in all the samples I've examined over the last few days, that for the **updateTime** and **validTimes** block... **validTimes** always starts 6hrs behind **updateTime** (same with the individual values)

Does that simply mean the forecast always has 6 hours of historical data?

I know that might be obvious to some, but I never like making an assumption for something like this

```
        "updateTime": "2022-12-20T23:15:59+00:00",
        "validTimes": "2022-12-20T17:00:00+00:00/P8DT2H",
        "elevation": {
            "unitCode": "wmoUnit:m",
            "value": 167.03039999999999
        },
        "forecastOffice": "https://api.weather.gov/offices/FWD",
        ...
        "temperature": {
            "uom": "wmoUnit:degC",
            "values": [
                {
                    "validTime": "2022-12-20T17:00:00+00:00/PT1H",
                    "value": 8.3333333333333339
                },
                {
                    "validTime": "2022-12-20T18:00:00+00:00/PT1H",
                    "value": 10
                ...
```

**Comment by sullynole** [comment, marked as answer]
updateTime is the last time any of the grids were published/updated to the world by the local office.
validTimes encompasses all of the times with available data for any of the grids.  There usually are grids populated with data up to ~6h in the past when the forecast is published by the forecaster at the local office.
